bazel/coverage/fuzz_coverage_wrapper.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -ex
+
+TEST_BINARY=$1
+shift
+
+mkdir -p fuzz_corpus/seed_corpus
+cp -r $@ fuzz_corpus/seed_corpus
+
+# Don't collect coverage when generating corpus
```

Comment:
Why not collect coverage in this 60 second run, and remove the run over the corpus in line 14?

Reply:
I followed what was happening before this PR. should we just collect coverage in 60 seconds run?

Reply:
yep!